src/ontology/imports/dental_material_import.owl

```diff
@@ -519,6 +521,24 @@ Changed definition to read that a &quot;portion of dental restoration material&q
     
 
 
+    <!-- http://purl.obolibrary.org/obo/imports/dental_material_import.owl#direct -->
+
+    <Class rdf:about="http://purl.obolibrary.org/obo/imports/dental_material_import.owl#direct">
```

Comment:
The approach taken by the OHD is different than yours. Materials are not classified a being direct or direct. Procedures are classified as being direct (e.g., a filling) or indirect (e.g., a crown).   
Some materials may be used in either indirect or direct procedure.

The OHD has high level classes for both indirect or direct procedures. See: 
- [direct restoration procedure](http://purl.obolibrary.org/obo/OHD_0000303)
- [indirect restoration procedure](http://purl.obolibrary.org/obo/OHD_0000304)

These are logically defined class that use the reasoner to infer the subclasses.  

Some of the procedure classes are related to specific restoration materials. E.g., The class [resin filling restoration procedure](http://purl.obolibrary.org/obo/OHD_0000042) is (logically) defined as being a procedure that uses (i.e., [has_specified_input](http://purl.obolibrary.org/obo/OBI_0000293) some [resin-based composite restoration material](http://purl.obolibrary.org/obo/OHD_0000036).